vamp-plugins/dsp/TempoTrack.cpp

```diff
@@ -544,7 +544,7 @@ int TempoTrack::findMeter(double *ACF, unsigned int len, double period)
 	tsig = 4;
     }
 
-
+    delete(dbf);

```

Comment:
It looks like dbf is unused. Can we delete it? 
We should provide a patch upstream as well.